src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/prison/UserController.kt

```diff
@@ -146,6 +149,63 @@ class UserController(
     createUserRequest: CreateUserRequest,
   ) = NewPrisonUserDto.fromDomain(prisonUserService.createUser(createUserRequest))
 
+  @PostMapping("/linkedprisonusers/admin", produces = [MediaType.APPLICATION_JSON_VALUE])
+  @PreAuthorize("hasRole('ROLE_CREATE_USER')")
+  @ResponseStatus(HttpStatus.CREATED)
+  @Operation(
+    summary = "Create a Linked DPS Admin user",
+    description = "Creates a specific Linked DPS user. Requires role ROLE_CREATE_USER",
```

Comment:
Wording doesn't feel quite right - possibly "Link an Admin User to an existing General Account" would be better